app/src/main/java/ch/hikemate/app/model/facilities/Facility.kt

```diff
@@ -0,0 +1,72 @@
+package ch.hikemate.app.model.facilities
+
+import org.osmdroid.util.GeoPoint
+
+/**
+ * Represents a facility or amenity point. Each facility has a specific type (e.g., toilet, parking)
+ * and geographical coordinates.
+ *
+ * @property type The type of facility, defined in [FacilityType]
+ * @property coordinates Geographic coordinates of the facility
+ */
+data class Facility(val type: FacilityType, val coordinates: GeoPoint)
```

Comment:
Question: Why would you not use `LatLong` as in the rest of the app ? This would be easier in order to have the same type everywhere.